packages/jest-docblock/src/index.js

```diff
@@ -47,5 +49,50 @@ function parse(docblock: string): {[key: string]: string} {
   return result;
 }
 
+function print(
+  object: {[key: string]: string} = {},
+  comments: string = '',
+): string {
+  const head = '/**';
+  const start = ' *';
+  const tail = ' */';
+
+  const keys = Object.keys(object);
+  const line = os.EOL;
+
+  const printedObject = keys
+    .map(key => start + ' ' + printKeyValue(key, object[key]) + line)
+    .join('');
+
+  if (!comments) {
+    if (keys.length === 0) {
+      return '';
+    }
+    if (keys.length === 1) {
+      return `${head} ${printKeyValue(keys[0], object[keys[0]])}${tail}`;
+    }
+  }
+
+  const printedComments =
+    comments
+      .split(os.EOL)
```

Comment:
can we detect the original `EOL` in the comment instead (https://github.com/sindresorhus/detect-newline)? People might use LF even if they're on Windows (e.g. airbnb eslint config requires LF regardless of OS)

Reply:
Should I do the same for the existing `parse`, which has a hard-coded `\n`?

Reply:
I think that makes sense, yeah 🙂